elasticsearch-evolution-core/src/test/java/com/senacor/elasticsearch/evolution/core/internal/migration/input/MigrationScriptReaderTest.java

```diff
@@ -0,0 +1,120 @@
+package com.senacor.elasticsearch.evolution.core.internal.migration.input;
+
+import com.senacor.elasticsearch.evolution.core.api.MigrationException;
+import com.senacor.elasticsearch.evolution.core.internal.model.migration.RawMigrationScript;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ * @author Andreas Keefer
+ */
+class MigrationScriptReaderTest {
+    @Nested
+    class readResources {
+
+        @Test
+        void fromClassPathResourcesDircetory() {
+            MigrationScriptReaderImpl reader = new MigrationScriptReaderImpl(Arrays.asList("classpath:scriptreader"),
+                    StandardCharsets.UTF_8,
+                    "c",
+                    Arrays.asList(".http"));
+            List<RawMigrationScript> actual = reader.read();
+            assertThat(actual).containsExactlyInAnyOrder(new RawMigrationScript().setFileName("content.http").setContent("content!"),
```

Comment:
so kannst du das z.B. besser lesbar strukturieren:
```suggestion
            assertThat(actual)
                    .containsExactlyInAnyOrder(
                            new RawMigrationScript()
                                    .setFileName("content.http")
                                    .setContent("content!"),
                            new RawMigrationScript()
                                    .setFileName("content_sub.http")
                                    .setContent("sub content!"));
```
gilt natürlich für alle anderen tests auch ;)